messages/teleporter/message.go

```diff
@@ -35,6 +35,7 @@ type TeleporterMessageReceipt struct {
 // in the contract deployed on the destination chain
 type ReceiveCrossChainMessageInput struct {
 	RelayerRewardAddress common.Address `json:"relayerRewardAddress"`
+	Index                uint32         `json:"index"`
```

Comment:
Nit: This is dependent on the corresponding Teleporter PR, but I think it would make more sense to put the `index` as the first parameter of `receiveCrossChainMessage` since it specifies which message is being received.

Reply:
updated to `messageIndex` to match teleporter PR, and made first parmeter